eng/doc/CrossPlatformCryptography.md

```diff
@@ -1,7 +1,7 @@
 # Cross-Platform Cryptography in Microsoft Go
 
 Cryptographic operations in Microsoft Go are delegated to the operating system (OS) libraries in some conditions described.
```

Comment:
This was also part of my earlier suggestion--I think I was originally going to continue with "described in the FIPS README" in the first sentence, but got distracted, ended up adding a new sentence instead, and forgot about this stray word. 😄 

```suggestion
Cryptographic operations in Microsoft Go are delegated to the operating system (OS) libraries in some conditions.
```